includes/data/connection/class-downloadable-item-connection-resolver.php

```diff
@@ -26,6 +27,32 @@ class Downloadable_Item_Connection_Resolver extends AbstractConnectionResolver {
 	 */
 	use WC_Connection_Functions;
 
+	const PREFIX = 'DI';
+
+	/**
+	 * get_loader_name
```

Comment:
Not sure why the name of the method is here.

Reply:
It's carry over from when I copied the signature from WPGraphQL's AbstractConnectionResolver class. I'll clean it up.